io.github.NickKarpowicz.LightwaveExplorer.json

```diff
@@ -0,0 +1,165 @@
+                    "md5": "560466f714a79fce3cb8793528d487c8"
+        
+                }
+            ]
+        },
+        {
+            "name": "fmt",
+            "config-opts": ["-DCMAKE_POSITION_INDEPENDENT_CODE=TRUE", "-DFMT_TEST=OFF"],
+            "buildsystem": "cmake",
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/fmtlib/fmt", 
+                    "tag": "10.1.0",
+                    "commit": "e57ca2e3685b160617d3d95fcd9e789c4e06ca88" } 
+                ]
+        },
+        {
+            "name": "gcem",
+            "buildsystem": "cmake",
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/kthohr/gcem", 
+                    "tag": "v1.17.0",
+                    "commit": "8422f5307b0498d09cf626f38acb03fe9f1efd94" } 
+                ]
+        },
+        {
+            "name": "dlib",
+            "buildsystem": "simple",
+            "build-commands":["cp -R dlib /app/include/"],
+            "sources": [ 
+                { 
+                    "type": "git", 
+                    "url": "https://github.com/davisking/dlib", 
+                    "tag": "v19.24.2",
+                    "commit": "39749fba5163d4e25030358f6aa8d91617bfdca2" } 
+                ]
+        },
+        {
+            "name": "cuda",
+            "buildsystem": "simple",
+            "build-commands":["mkdir /app/tmp","sh cuda_12.2.1_535.86.10_linux.run --silent --toolkitpath=/app/opt/cuda --toolkit --override --tmpdir=/app/tmp"],
+            "sources":[
+                {
+                    "type": "file",
+                    "url": "https://developer.download.nvidia.com/compute/cuda/12.2.1/local_installers/cuda_12.2.1_535.86.10_linux.run",
+                    "md5": "2579fcb61cd80d3284d73da93785550f"
+                }
+            ]
+        },
+        {
+            "name": "LightwaveExplorer",
+            "buildsystem": "simple",
+            "build-commands": ["touch localscript.sh",
+            "echo \". /app/opt/intel/oneapi/setvars.sh\" >> localscript.sh",
+            "echo \"export PATH=$PATH:/app/opt/cuda/bin\" >> localscript.sh",
+            "echo \"export CMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH:/app/opt/intel/oneapi/mkl/latest/lib/cmake/mkl\" >> localscript.sh",
+            "echo \"export CUDA_PATH=$PATH:/app/opt/cuda/\" >> localscript.sh",
+            "echo \"cmake -DMAKECUDAFLATPAK=TRUE -DCMAKE_CUDA_ARCHITECTURES=75 .\" >> localscript.sh",
+            "echo \"make\" >> localscript.sh",
+            "chmod +x localscript.sh","./localscript.sh",
+            "cmake --install . --prefix \"/app\""],
+            "build-options": {
+                "cxxflags": "-O3 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong -grecord-gcc-switches",
```

Comment:
do you really need these flags? Most are set by default.

Reply:
These are essentially the default flags, but I need to remove D_GLIBCXX_ASSERTIONS, which throws an exception when a std::complex NaN is encountered, before the simulation's own error-handling deals with it. This causes the program to throw a not-very-informative assertion failure rather than an in-GUI message giving hints at how to fix the input parameters.